e2e/connect/input/demo.nomad

```diff
@@ -0,0 +1,61 @@
+ job "countdash" {
+   datacenters = ["dc1"]
+
+   group "api" {
+     network {
+       mode = "bridge"
+     }
+
+     service {
+       name = "count-api"
+       port = "9001"
+
+       connect {
+         sidecar_service {}
+       }
+     }
+
+     task "web" {
+       driver = "docker"
+       config {
+         image = "hashicorpnomad/counter-api:v1"
+       }
+     }
+   }
+
+   group "dashboard" {
+     network {
+       mode ="bridge"
+       port "http" {
```

Comment:
We don't really need this port stanza for the test

Reply:
I want to keep it identical to the guide.